.github/workflows/auto-publish.yml

```diff
@@ -0,0 +1,20 @@
+# .github/workflows/auto-publish.yml
+name: CI
+on:
+  pull_request: {}
+  push:
+    branches: [main]
+jobs:
+  main:
+    name: Build, Validate and Deploy
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v3
+      - uses: w3c/spec-prod@v2
+        with:
+          TOOLCHAIN: respec
+          W3C_ECHIDNA_TOKEN: ${{ secrets.W3C_TR_TOKEN }}
+          W3C_WG_DECISION_URL: https://www.w3.org/2017/vc/WG/Meetings/Minutes/2023-04-12-vcwg#resolution1
+          W3C_BUILD_OVERRIDE: |
+             shortName: vc-jwt
+             specStatus: WD
```

Comment:
do i need to update the index.html to this as well?

Reply:
You mean setting the specStatus to WD? No. Echidna, more exactly the respec engine does that for you (it is possible to start up the respec transform engine with query parameters in the URL to overwrite virtually any respec configuration value).